### DISCUSSION TITLE
tmp folder replaced inside the build folder after new deployment.

### DISCUSSION THREAD

**Discussion by mahbubhasanhira** [adonisjs/core#4517] [Help]
I have a v5 API backend and I am using the local tmp folder for media. But when I deploy every time it generates a new build folder and removes the previous tmp folder with all media files from the new build folder. Is there any way to copy the tmp folder from the build folder before the new deployment and then it again move/copy the tmp folder from the parent or copy where it was. I want to automate the process with a bash command or something like that. and I am using Cleavr to deploy the app.

**Comment by thetutlage** [comment]
You should keep the user uploaded files outside of the application codebase. Cleavr guys have this forum thread on the same. https://forum.cleavr.io/t/linking-to-a-shared-folder-across-deployments/184
